pointmatcher/ErrorMinimizersImpl.h

```diff
@@ -76,11 +76,22 @@ struct ErrorMinimizersImpl
 		virtual T getOverlap() const;
 	};
 
+  	struct PointToPointSimilarityErrorMinimizer: ErrorMinimizer
+	{
+		inline static const std::string description()
+		{
+			return "Point-to-point similarity error (rotation + translation + scale). Based on SVD decomposition. Based on \\cite{Umeyama1991}.";
```

Comment:
If I understood well, the scale computed here is equal for all dimensions. The term similarity is appropriate here, but extra explanation would avoid users confusing it with affine.